src/iop/colorchecker.c

```diff
@@ -109,6 +109,8 @@ typedef struct dt_iop_colorchecker_global_data_t
   int kernel_colorchecker;
 } dt_iop_colorchecker_global_data_t;
 
+#define SQR(x) ((x) * (x))
```

Comment:
(just a minor note as I happened to stumble upon this): there is a widely used `static inline float sqf(const float x)` function in `src/develop/openmp_maths.h` – could that one be used here?

Reply:
Thank you, yes!

Reply:
Not sure maybe a commit is missing. This is marked as resolved, you agreed to use sqf() already defined but I still see SQR() macco used.

Reply:
It should be there in 57d8efd, following this conversation? It seems to have integrated, in https://github.com/darktable-org/darktable/pull/9892/files I see only `sqf()`, not `SQR()`.